### PR TITLE
Used Live performance test count

### LINKED ISSUE #1705: [Meshery] Use live performance test count
**Current Behavior**
The Meshery project page (https://layer5.io/service-mesh-management/meshery) counts up to a static number (2,000+).

**Desired Behavior**
Use the public endpoint (https://meshery.layer5.io/result/total) that has a real-time test result count instead.

See an example of its use here - https://meshery.io

The animated count should still occur, but count to the actual number instead of a hardcoded number.

 **Screenshots / Mockups**
Screen Shot 2021-03-16 at 12 18 23 PM

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by chinmaym07**
Signed-off-by: chinmaym07 <b418020@iiit-bh.ac.in>

**Description**

This PR fixes #1705 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 88c7c224934644e157149cd185583337f49b53e1 at: https://6051d4063582c470431150f2--layer5ng.netlify.app

File: src/sections/Meshery/Features-Col/data/index.js [hunk 1]
```diff
@@ -80,7 +79,7 @@ const data = {
         }
       ],
       count: {
-        value: 2000,
+        value: null,
```
Comment (Nikhil-Ladha) [inline src/sections/Meshery/Features-Col/data/index.js:82]: `null` is of `object` type in JS. Better, to use `0` here.

File: src/sections/Meshery/Features-Col/index.js [hunk 2]
```diff
@@ -42,7 +54,7 @@ const Features = () => {
                     <Counter
                       duration={6}
                       separator=","
-                      end={feature.count.value}
+                      end={feature.count.value !== null ? feature.count.value:performanceCount}
```
Comment (Nikhil-Ladha) [inline src/sections/Meshery/Features-Col/index.js:57]: Update this as well, as per the above comment.
Also, add improve code readability around `:` (else condition)

**Review by Nikhil-Ladha — APPROVED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 17e64c58057c916174d8db6daab4d6719ccde280 at: https://6051fc0acae19288c5c39fa5--layer5ng.netlify.app

**Pull request merged**